common-app-lib/src/cmr/common_app/services/kms_fetcher.clj

```diff
@@ -112,9 +100,8 @@
 
 (defn refresh-kms-cache-job
   "The singleton job that refreshes the KMS cache. The keywords are infrequently updated by the
-  GCMD team. They update the CSV file which we read from every 6 hours. I arbitrarily chose 2 hours
-  so that we are never more than 8 hours from the time a keyword is updated."
+  GCMD team, usually once a week."
   [job-key]
   {:job-type RefreshKmsCacheJob
    :job-key job-key
-   :interval 7200})
+   :daily-at-hour-and-minute [02 00]})
```

Comment:
we may need a wiki page to track the schedules for all our jobs to make sure we spread them out, just thinking out loud